.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -40,17 +33,15 @@ assignees: ''
 - [ ] Merge PR
 - [ ] Create and push *both* tags to GitHub (`vX.Y.Z`, `X.Y.Z`)
   - Pull latest branch locally and run `contrib/release/tag-release.sh`
-- [ ] Wait for docker builds to complete
-  - [cilium](https://hub.docker.com/repository/docker/cilium/cilium/builds)
-  - [operator](https://hub.docker.com/repository/docker/cilium/operator/builds)
-  - [docker-plugin](https://hub.docker.com/repository/docker/cilium/docker-plugin/builds)
-  - [operator-generic](https://hub.docker.com/repository/docker/cilium/operator-generic/builds)
-  - [operator-aws](https://hub.docker.com/repository/docker/cilium/operator-aws/builds)
-  - [operator-azure](https://hub.docker.com/repository/docker/cilium/operator-azure/builds)
-  - [hubble-relay](https://hub.docker.com/repository/docker/cilium/hubble-relay/builds)
-  - [clustermesh-apiserver](https://hub.docker.com/repository/docker/cilium/clustermesh-apiserver/builds)
+- [ ] Ask a maintainer to approve the build in the following links:
+  - [Cilium v1.10](https://github.com/cilium/cilium/actions?query=workflow:%22Image+Release+Build%22)
+  - [Cilium v1.9](https://github.com/cilium/cilium/actions?query=workflow:%22Image+Release+Build+v1.9%22)
+  - [Cilium v1.8](https://github.com/cilium/cilium/actions?query=workflow:%22Image+Release+Build+v1.8%22)
+  - [Cilium v1.7](https://github.com/cilium/cilium/actions?query=workflow:%22Image+Release+Build+v1.7%22)
   - Check if all docker images are available before announcing the release
     `make -C install/kubernetes/ check-docker-images`
+- [ ] Get the image digests from the build process (click in the GH Run > "Display Digests"),
+      update the digests in the helm charts locally and make a commit / PR with the digests.
```

Comment:
Is this to expose the new image version on the v1.X branch? Maybe we should push this out to the post-release steps?

Though we also want to use this process to do the equivalent of the `contrib/release/pull-docker-manifests.sh` step below, just to share the digests as part of the release notes.

Reply:
oh, no, it's for the *helm* charts, like the text says right there. I look forward to the script to automate this, this sounds easy to copy+paste the wrong thing in the wrong place.

Reply:
hehe, trying again. So this is for generating the helm charts, but using a commit on the Cilium repo. The fact that Cilium v1.X.Y tag will have helm charts pointing towards the v1.X.Y-1 sha is just a poor side-effect. What we're trying to achieve is to embed the correct shas into the helm charts for the actual release 1.X.Y charts.

Reply:
I don't think I actually explained that well, but one observation is: "Don't run `git checkout v1.8.8; helm install ./install/kubernetes/cilium` and expect v1.8.8 to be installed, this will *not* do what you expect"

The other observation is just that copying and pasting random shas around the place is error-prone, but we should be able to come up with a relatively simple workflow to pull the digests probably from the most recent Workflow (via the [links above](https://github.com/cilium/cilium/actions?query=workflow:%22Image+Release+Build+v1.9%22)) and apply those shas to the tree.

Reply:
Along this line of thinking, why can't we arbitrarily trigger the images to be build from a specific git sha we select *first*, then do the whole "prepare for v1.X.Y release" commit preparation, author updates, helm chart updates, git tags, etc. based on top of those shas? Is it just because you have to push a tag first to trigger the workflow request to show up in github?

Reply:
oh, I guess the version number that shows up inside the image will reflect whatever was in the tree of the previous commit.

Reply:
no... let's recap... a) this will work fine and as expected:
```
git checkout v1.8.8; helm install ./install/kubernetes/cilium
```

b) this will also work fine and as expected:
```
git checkout v1.8.8 +1 (the next commit of v1.8.8); helm install ./install/kubernetes/cilium
```

the difference between a) and b) is that a) points to image tags where b) points to image digests. Which we can only have access **after** tagging and build images. The commit b) will be the one used to generate the helm charts.